src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -182,6 +186,8 @@ export const getFormSchema = (
           patch: {
             type: "object" as "object",
             title: "Patch Settings",
+            description:
+              "Sets if users are allowed to create patches for this branch.",
```

Comment:
<img width="581" alt="image" src="https://user-images.githubusercontent.com/13104717/169374315-9b0dad20-5279-4838-a1f8-2d8765125234.png">
now that we have this description, do we still need the **Patching** subtitle?